docs/volunteering/glossary.md

```diff
@@ -59,6 +59,10 @@ _SR_:
 
 : an abbreviation for 'Student Robotics'
 
+[_SR(A)WN_](./srawn.md):
```

Comment:
Just a thought -- maybe worth also including here the spelling without the parens (i.e: `SRAWN`) in case people have only heard the name and not seen it written? While I wouldn't normally suggest keyword stuffing to aid with alternate spellings, I think this one is sufficiently unusual to be worth it.